test/e2e/egress_test.go

```diff
@@ -402,10 +402,17 @@ func testEgressCRUD(t *testing.T, data *TestData) {
 				return true, nil
 			})
 			require.NoError(t, err, "Expected egressIP=%s nodeName in %s, got egressIP=%s nodeName=%s", tt.expectedEgressIP, sets.List(tt.expectedNodes), egress.Spec.EgressIP, egress.Status.EgressNode)
+
 			if egress.Status.EgressNode != "" {
 				exists, err := hasIP(data, egress.Status.EgressNode, egress.Spec.EgressIP)
 				require.NoError(t, err, "Failed to check if IP exists on Node")
 				assert.True(t, exists, "Didn't find desired IP on Node")
+				// Testing the events recorded during creation of an Egress resource.
+				expectedMessage := fmt.Sprintf("Assigned egress %s with IP %s on node", egress.Name, tt.expectedEgressIP)
+				fieldSelector := fmt.Sprintf("involvedObject.name=%s", egress.Name)
+				events, err := data.clientset.CoreV1().Events("").List(context.TODO(), metav1.ListOptions{FieldSelector: fieldSelector})
```

Comment:
can we use `c.CoreV1().Events("").Search`?

Reply:
sure we can use this